tools/StaticAnalysis/Exceptions/Az.NetworkCloud/SignatureIssues.csv

```diff
@@ -14,3 +14,56 @@
 "Az.NetworkCloud","New-AzNetworkCloudVirtualMachinePlacementHintObject","New-AzNetworkCloudVirtualMachinePlacementHintObject","1","8100","New-AzNetworkCloudVirtualMachinePlacementHintObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Az.NetworkCloud","New-AzNetworkCloudIpAddressPoolObject","New-AzNetworkCloudIpAddressPoolObject","1","8100","New-AzNetworkCloudIpAddressPoolObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Az.NetworkCloud","New-AzNetworkCloudBareMetalMachineConfigurationDataObject","New-AzNetworkCloudBareMetalMachineConfigurationDataObject","1","8100","New-AzNetworkCloudBareMetalMachineConfigurationDataObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Az.NetworkCloud","Get-AzNetworkCloudAgentPool","Get-AzNetworkCloudAgentPool","2","8010","Get-AzNetworkCloudAgentPool Changes the ConfirmImpact but does not set the SupportsShouldProcess property to true in the cmdlet attribute.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
Where were these signature issues from? Were they warnings or errors? If warning, then please remove them and no need to suppress.